File: DecimaldenBinaryeCevirme/DecimaldenBinaryeCevirme.java
package decimaldenbinaryecevirme;

import java.util.Scanner;

public class DecimaldenBinaryeCevirme {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Dönüşüm yapılacak sayıyı girin : ");
        int decimalSayi = reader.nextInt();
        String binarySayi = "";
        if (decimalSayi == 0) {
            binarySayi = "0";
        }
        while (decimalSayi != 0) {
            binarySayi = decimalSayi % 2 + binarySayi;
            decimalSayi /= 2;
        }
        System.out.println(binarySayi);
    }
}

File: DecimaldenBinaryeCevirme/DecimaldenBinaryeCevirme.py
def binary_donusum(decimal_sayi):

    binary_sayi = ''

    while decimal_sayi != 0:

        binary_sayi = str(decimal_sayi % 2) + binary_sayi
        decimal_sayi = decimal_sayi / 2
        decimal_sayi = int(decimal_sayi)

    return binary_sayi

print(binary_donusum(int(input("Bir sayı girin : "))))
